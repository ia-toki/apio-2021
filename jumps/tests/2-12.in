209636825ecd2c38299498cd26890ef71dd128c2
200 200
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 126 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 187 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 41 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143 144 145 146 147 148 149 150 151 152 153 154 155 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 174 175 176 177 178 179 180 181 182 183 184 185 186 73 188 189 190 191 192 193 194 195 196 197 198 199 200
0 0 199 199
108 174 187 195
8 35 42 133
23 96 137 173
28 74 129 197
50 85 106 173
69 183 199 199
0 11 120 140
15 21 152 180
22 43 107 138
39 115 195 198
19 42 128 184
0 3 116 155
38 43 44 136
115 176 177 188
20 21 137 173
44 44 66 146
10 19 57 171
43 154 156 166
9 60 83 165
102 137 138 183
6 18 113 179
48 58 178 198
45 88 96 153
3 10 38 197
67 173 188 189
49 87 93 103
142 195 199 199
163 174 197 199
136 163 199 199
63 95 192 198
8 23 94 110
189 198 199 199
1 28 129 157
3 160 192 194
7 53 154 173
31 87 145 187
28 107 158 191
8 35 123 182
4 51 146 197
102 164 180 181
18 29 93 118
70 196 197 199
37 89 114 132
73 127 157 169
72 197 199 199
55 114 142 153
24 33 178 188
17 125 147 169
51 86 190 192
42 58 122 142
76 111 127 179
3 7 195 196
52 88 100 157
168 186 196 197
26 114 157 183
98 173 192 193
60 183 184 189
78 196 199 199
32 87 169 178
174 180 190 194
35 131 158 176
114 177 194 195
33 145 192 194
72 148 183 187
44 163 172 180
93 187 198 199
86 158 170 197
75 86 142 179
39 181 196 197
49 59 97 165
35 48 135 162
40 56 98 114
118 192 193 193
60 63 104 183
29 43 65 115
115 166 189 198
12 15 172 178
83 174 179 191
1 1 122 145
40 66 163 165
12 21 131 182
98 110 191 193
8 182 189 190
20 139 141 197
12 24 155 157
35 41 45 60
10 133 156 182
9 32 41 121
31 42 91 99
93 170 197 197
128 168 192 195
23 38 50 199
132 140 191 195
149 168 177 193
19 44 46 87
158 162 185 194
6 102 163 182
14 19 147 198
118 160 177 192
73 122 127 156
4 27 32 58
93 97 167 187
30 128 133 180
66 192 193 196
145 169 194 195
0 28 163 192
38 63 162 169
32 91 157 161
18 46 144 166
67 118 188 192
24 34 163 183
45 48 116 122
39 61 119 151
25 33 101 199
44 169 190 195
39 87 96 114
8 47 178 187
124 167 176 177
26 108 190 197
19 32 165 175
1 14 174 177
3 194 197 197
26 39 115 157
45 72 148 179
55 136 170 172
24 53 67 187
68 90 116 151
12 168 190 191
110 187 199 199
4 6 40 148
32 50 127 195
78 192 193 194
66 191 195 196
43 76 82 122
4 16 106 194
46 185 189 192
10 17 99 107
48 105 194 195
0 32 34 87
34 127 156 191
57 163 167 170
21 193 194 199
81 112 120 131
72 137 174 179
25 30 170 174
51 75 161 189
119 171 181 194
10 73 112 184
88 169 173 179
8 196 199 199
107 170 189 193
128 148 182 192
21 87 98 159
1 159 179 194
112 166 186 192
119 119 129 167
13 122 194 195
2 198 199 199
16 69 110 171
89 104 127 164
25 150 192 192
137 189 196 197
103 108 196 196
26 28 150 169
61 79 111 119
31 53 79 148
23 103 130 194
90 154 199 199
23 24 36 141
28 114 131 135
67 160 167 184
75 107 153 166
54 84 142 173
82 127 141 196
3 18 135 194
147 166 192 194
37 71 95 139
0 170 173 190
91 177 183 188
56 70 115 163
136 193 199 199
146 148 164 173
79 165 168 188
41 49 139 140
2 54 165 180
32 46 115 186
30 44 134 153
1 3 35 94
10 33 185 190
17 48 89 198
48 50 192 197
42 74 186 186
32 160 166 178
36 96 184 184
123 155 194 195
23 108 153 189
20 81 185 196
135 141 197 198
69 79 145 195
