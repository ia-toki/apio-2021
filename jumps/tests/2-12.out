fc5005cd102fffc504bc1615010ebf88823d65f3
OK
54
2
6
40
1
21
13
-1
-1
-1
9
-1
-1
1
1
-1
2
22
2
-1
1
-1
3
8
28
2
6
4
12
14
6
-1
1
-1
6
-1
1
1
-1
2
16
-1
1
25
30
2
27
9
22
4
-1
15
43
12
10
42
7
1
3
81
5
27
9
6
1
9
11
2
55
10
-1
-1
-1
1
-1
1
4
-1
2
-1
-1
-1
6
3
1
-1
4
23
9
-1
12
7
3
6
2
1
2
60
23
2
4
5
2
5
1
9
14
-1
65
-1
2
-1
-1
-1
9
4
9
2
9
4
-1
-1
3
-1
-1
34
1
26
4
12
34
2
1
4
6
26
3
-1
8
2
1
4
1
8
37
-1
1
2
39
4
3
4
2
11
1
2
9
8
1
-1
22
6
7
11
-1
32
-1
1
14
12
16
7
45
57
2
24
7
-1
1
2
-1
6
16
2
-1
-1
-1
-1
32
9
2
8
-1
6
87
9
1
1
12
1
